hw2/04-background.html

```diff
@@ -6,23 +6,38 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
+  <div id="background">
```

Comment:
just a suggestion, instead of covering the body with a div, you can straight away add an id to the body

Reply:
good idea